documents/policies/external/Competition Requirements.md

```diff
@@ -104,7 +104,8 @@ The WCA Delegate must submit the following information when requesting approval
    3. The competitor limit is per competition or per event. Per day competitor limits are not allowed.
       1. Exception: for 3x3x3 Fewest Moves simultaneous competitions, the competitor limit is per location.
    4. The competitor limit is not greater than half the maximum number of people allowed at public gatherings in the competition venue.
-   5. The WCA Competition Announcement Team will review the competitor limit and/or qualification results or rankings to ensure that they are utilized properly.
+      1. If such a maximum number is not available, the best estimate from a Delegate may be used.
+   6. The WCA Competition Announcement Team will review the competitor limit and/or qualification results or rankings to ensure that they are utilized properly.
```

Comment:
This should still be 5. not 6. right?

Reply:
Yup - thanks for the catch! Not entirely sure why it changed, but it should be fixed now.